feature-group-definitions/bigint.yml

```
@@ -1,4 +1,5 @@
 name: BigInt
+description: BigInt is a built-in JavaScript type that represents numeric values which are too large to be represented by the Number type.
```

Trying to fit this into the "The $syntax $technology $typeofthing" pattern like "The `aspect-ratio` CSS property ..." How about this:

```suggestion
description: The `BigInt` JavaScript type represents integers of any size, including integers too large for the primitive `Number` type.
```